frontend/src/screens/SearchGif.svelte

```diff
@@ -62,6 +70,54 @@
         }, 300);
     };
 
+    const throwGiphyError = async (e: gifFetchError) => {
+
+        Swal.fire({
+            icon: 'error',
+            titleText: 'Whoops :(',
+            background: '#181818',
+            color: 'white',
+            text: `It seems like Giphy doesn't work for you right now.
+                    Would you like to report this bug and help us improve GYF?`,
+            showCancelButton: true,
+            cancelButtonText: 'No',
+            cancelButtonColor: 'red',
+            confirmButtonText: 'Yes',
+            confirmButtonColor: 'green'
```

Comment:
I think we shouldn't "force" the user to report errors by setting the cancel button's color to red.
We should make the Confirm-Button a blue shade and the cancel button grey

```suggestion
            cancelButtonText: 'No',
            cancelButtonColor: '#95a5a6',
            confirmButtonText: 'Yes',
            confirmButtonColor: '#3498db'
```